lp-build-core.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+
```

Comment:
If this is python 2.x can you please add

```
from __future__ import absolute_import, print_function
```

If this is python3 then please adjust the first line to read `#!/usr/bin/python3`